common/config/rush/pnpm-lock.yaml

```diff
@@ -1132,6 +1132,29 @@ packages:
       - supports-color
     dev: false
 
+  /@azure/service-bus/7.4.0:
```

Comment:
which part of our repo depend on this version?

Reply:
eventgrid and perf-service-bus depends on `^7.0.0`.

Reply:
I see. It may not be worth it to make perf-service-bus depend on this beta but perhaps something to consider.

Reply:
I looked at history and we haven't done that for older versions.  Though we probably want perf tests to always use source version @HarshaNalluru?